NEXT_CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 ## Release v0.267.0
 
 ### Notable Changes
+* Add 'databricks bundle plan' command. This command shos the deployment plan for the current bundle configuration without making any changes. ([#3530](https://github.com/databricks/cli/pull/3530)
```

Comment:
```suggestion
* Add 'databricks bundle plan' command. This command shows the deployment plan for the current bundle configuration without making any changes. ([#3530](https://github.com/databricks/cli/pull/3530)
```

Bonus points if we can link to docs here. (And we should make sure we start the docs process for this if that hasn't already.)